helpers/networker.py

```diff
@@ -57,15 +57,26 @@ def change_gro(self, dev, on=True):
     def change_gso(self, dev, on=True):
         self._set_state(dev, "gso", on)
 
-    def run_test_tso_gro_gso(self, client, server, test, mtu, tso, gro, gso):
+    def get_tcp_option(self, option_name):
+        cmd = f"sudo sysctl {option_name}"
+        out = remote.client.run_cmd(cmd)
```

Comment:
Why do you use `remote.client` here? Probably, should it use as method argument? Do we should change options for TempestaFW VM (remote setup)?

Reply:
remote.tempesta instead according discussion

Reply:
Yes fixed